.vscode/settings.json

```diff
@@ -30,18 +30,18 @@
     "certa.json": "jsonc"
   },
   "files.exclude": {
-    "**/.nyc_output": true,
-    "**/.rush": true,
-    "**/*.build.log": true,
-    "**/*.build.error.log": true,
-    "**/generated-docs": true,
-    "**/staging-docs": true,
-    "**/package-deps.json": true,
-    "**/node_modules": true,
-    "**/test-apps/**/build": true,
-    "**/out": true,
-    "**/lib": true,
-    "common/temp": true
+    "**/.nyc_output": false,
```

Comment:
revert this file
the extensions i shared the other day is a quick way to flip these values,
but i wouldnt commit the changes it made to your pr

Reply:
reverted